ui/main/src/app/modules/monitoring/components/monitoring-table/monitoring-table.component.ts

```diff
@@ -329,7 +329,7 @@ export class MonitoringTableComponent implements OnChanges, OnDestroy {
     export(): void {
         this.exportCancelled = false;
         // if monitoring has a specific configuration
-        if (this.monitoringConfig && this.monitoringConfig.export && this.monitoringConfig.export.fields) {
+        if (this.monitoringConfig.export?.fields) {
```

Comment:
this.this.monitoringConfig?.export?.fields

Reply:
resolved